packages/components/src/components/page-error/page-error.jsx

```diff
@@ -49,31 +52,66 @@ const PageError = ({
                         'dc-page-error__message-wrapper--left': !!image_url,
                     })}
                 >
-                    <span
+                    <Text
+                        color='prominent'
                         className={classNames('dc-page-error__message', {
                             'dc-page-error__message--left': !!image_url,
                         })}
                     >
                         {messages.map((message, index) =>
                             message.has_html ? (
-                                <p
+                                <Text
+                                    as='p'
+                                    size='s'
+                                    lineHeight='x'
```

Comment:
Based on latest base branch, this needs to be updated:
```suggestion
                                    line_height='x'
```

Reply:
Is there a reason you are changing `x` to `s`? Do they refer to the same line height?

Reply:
Ok, changed to new version